pkgs/jnigen/example/in_app_java/lib/android_utils.dart

```diff
@@ -188,7 +188,7 @@ class EmojiCompat_Config extends jni.JObject {
   static final _setReplaceAll = ProtectedJniExtensions.lookup<
           ffi.NativeFunction<
               jni.JniResult Function(ffi.Pointer<ffi.Void>, jni.JMethodIDPtr,
-                  ffi.VarArgs<(ffi.Int64,)>)>>("globalEnv_CallObjectMethod")
+                  ffi.VarArgs<(ffi.Uint8,)>)>>("globalEnv_CallObjectMethod")
```

Comment:
Just sanity-checking that we are going from 64-bit to 8-bit.

Reply:
We're just not defaulting to 64 anymore to fix the mac issue. Other architectures should not have any problem.